packages/synapse-react-client/src/utils/hooks/useOneSageURL.ts

```diff
@@ -31,11 +36,11 @@ export function getOneSageBaseUrl(
   }
 
   if (currentHostname === 'localhost' || currentHostname === '127.0.0.1') {
-    return `http://${currentHostname}:3000/`
+    return `http://${currentHostname}:3000`
```

Comment:
Just to make consistent with the other URLs that are returned (ONE_SAGE_STAGING_URL and ONE_SAGE_PRODUCTION_URL found in SynapseConstants, which do not have a trailing forward slash)